src/process.rs

```diff
@@ -200,7 +200,8 @@ impl Process {
                 .level_for("tokio_reactor", LevelFilter::Info)
                 .level_for("hyper", LevelFilter::Info)
                 .level_for("reqwest", LevelFilter::Info)
-                .level_for("h2", LevelFilter::Info);
+                .level_for("h2", LevelFilter::Info)
```

Comment:
These seem like unrelated changes, do they relate to #456? Do they need mentioning in the changelog? Should they be separated out from this PR?

Reply:
Given that sled was only introduced in #456, I don’t think we need to mention changes made to changes initially described in the changelog for the same release.

This should probably have been a separate PR but it is just a one liner.